packages/cysync-core/src/components/Topbar.tsx

```diff
@@ -97,6 +97,11 @@ const TopbarComponent: FC<TopbarProps> = props => {
       isLockscreenLoading={isLockscreenLoading}
       toggleDiscreetMode={() => dispatch(toggleDiscreetMode())}
       onSyncClick={onSyncClick}
+      tooltipText={
+        accountSync.syncState === AccountSyncStateMap.failed
+          ? 'Network Error'
```

Comment:
- The error msg should be defined in `i18n/en.ts`.
- Store the error message in the store so that we can display different error messages as needed

Reply:
changed